spec/uuid_helper_spec.coffee

```diff
@@ -0,0 +1,19 @@
+describe 'UUID', ->
+  before (done) ->
+    require ['helpers/uuid_helper'], (UUID) =>
+      @UUID = UUID
+      done()
+
+  describe 'API', ->
+    beforeEach ->
+      @subject = @UUID
+
+    it 'responds to generate', ->
+      expect(@subject).to.respondTo('generate')
+
+  describe '.generate', ->
+    it 'generates a UUIDv4-like string', ->
+      expect(@UUID.generate()).to.match(/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i)
```

Comment:
We are using both `@subject` and `@UUID`, let's choose one 😃 

if you decide to use `@UUID` remove the `beforeEach ->`